model/testresult/testresult.go

```diff
@@ -138,7 +138,7 @@ func (tr TestResult) GetLogURL(env evergreen.Environment, viewer evergreen.LogVi
 			}
 		}
 
-		return fmt.Sprintf("%s/test/%s/%d/%s?shareLine=%d", parsleyURL, tr.TaskID, tr.Execution, tr.GetLogTestName(), tr.LineNum)
+		return fmt.Sprintf("%s/test/%s/%d/%s?shareLine=%d", parsleyURL, tr.TaskID, tr.Execution, tr.TestName, tr.LineNum)
```

Comment:
It looks to me like the rest of the changes in this PR are meant to go with https://github.com/evergreen-ci/cedar/pull/520 and that this is the only change related to the PR description. Did you mean for all the rest of the changes to be in this PR?

Reply:
Yeah I need to update the description, sorry about that!

Reply:
Done.